### PR TITLE
Add TorchScript Docs Integrations Page

### PR DESCRIPTION
**Pull request by abirami-vina**
## 🛠️ PR Summary

<sub>Made with ❤️ by [Ultralytics Actions](https://github.com/ultralytics/actions)<sub>

### 🌟 Summary
Added TorchScript integration documentation for Ultralytics YOLOv8 model export.

### 📊 Key Changes
- 📄 Created new documentation (`torchscript.md`) for exporting Ultralytics YOLOv8 models to TorchScript format.
- 🚀 Added TorchScript to the list of integrations in the `mkdocs.yml` and in the integrations index (`index.md`).
- 🌐 Slight modifications to existing codebase for clarity and styling, such as clearer error messages in `dash.py` and styling fixes in `exporter.py`.

### 🎯 Purpose & Impact
- 📈 **Enables broader deployment:** The addition of TorchScript documentation allows users to export YOLOv8 models for use in various environments, expanding the deployment possibilities.
- 🔄 **Flexibility and performance:** TorchScript provides an optimized and serialized format suitable for platforms without Python, leading to potential performance improvements in inference tasks.
- ✨ **Enhanced user guidance:** Clear documentation will help users understand how to export their models and leverage them in production effectively.

### CODE REVIEW

**Comment by abirami-vina** [comment]
@glenn-jocher I've opened this pull request, but I'm going to double-check with Irantzu on better options for images as well.

**Comment by glenn-jocher** [comment]
@abirami-vina thanks for the update! It's great to hear you're exploring better options for images with Irantzu. If you need any assistance or have questions about integrating images into your pull request, feel free to reach out. Happy to help! 😊

File: docs/en/integrations/torchscript.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+!!! Tip "Installation"
+
+    === "CLI"
+    
+        ```bash
+        # Install the required package for YOLOv8
+        pip install ultralytics
+        ```
+
+For detailed instructions and best practices related to the installation process, check our [YOLOv8 Installation guide](../quickstart.md). While installing the required packages for YOLOv8, if you encounter any difficulties, consult our [Common Issues guide](../guides/yolo-common-issues.md) for solutions and tips.
```
Comment (Burhan-Q) [inline docs/en/integrations/torchscript.md:78]: Same thoughts here as mentioned in [the TFLite integration PR](https://github.com/ultralytics/ultralytics/pull/8522#discussion_r1507561616)

**Review by Burhan-Q — COMMENTED** [review]
Overall looks good!

File: docs/en/integrations/torchscript.md (hunk 1 above)
Comment (abirami-vina) [reply docs/en/integrations/torchscript.md:78]: Kk, thanks. I'll make edits accordingly.

**Comment by abirami-vina** [comment]
@Burhan-Q I've made the edits. Thanks.

**Comment by Burhan-Q** [comment]
Looks good to me! Thank you @abirami-vina 🎉

**Comment by glenn-jocher** [comment]
@abirami-vina looks good!

**Comment by glenn-jocher** [comment]
@abirami-vina @Burhan-Q PR merged, thanks guys!

**Pull request merged**